modules/local/merge_predictions/templates/merge_predictions.py

```diff
@@ -0,0 +1,308 @@
+#!/usr/bin/env python
+
+import argparse
+import shlex
+from enum import Enum
+import sys
+import typing
+import math
+
+import pandas as pd
+import mhcgnomes
+
+# Create logger object with date and time
+import logging
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s - %(name)s - %(levelname)s - %(message)s",
+    datefmt="%Y-%m-%d %H:%M:%S",
+)
+
+class PredictorBindingThreshold(Enum):
```

Comment:
Maybe we add references for these values. Also it feels like the wrong place for this functionality given the script name. Maybe something like `evaluate_predictions.py`. Just a minor thing though.